opm/grid/cpgrid/CpGridData.cpp

```diff
@@ -217,9 +217,9 @@ struct GetRowType<std::vector<E,A> >
 };
 
 PartitionType getPartitionType(const PartitionTypeIndicator& p, const EntityRep<1>& f,
-                               const CpGridData&)
+                               const CpGridData& grid)
 {
-    return p.getPartitionType(f);
+    return p.getPartitionType(Entity<3>(grid, f.index(), f.orientation()));
```

Comment:
Typo in the codim? for face, 1, for point 3. Same as below, just wondering if we also need some refactor in Entity to support a few methods there for Entity<1> for corner point grid

Reply:
Good catch. I'll fix that.